src/components/tasks/task-dialog.tsx

```diff
@@ -180,45 +246,82 @@ const TaskDialog = ({ projectId, milestoneId, open, task, onClose }: Props) => {
       name: "",
       startDate: null,
       endDate: null,
-      assignees: [],
       status: TaskStatus.NotStarted,
-      type: "",
-      estimatedDuration: "",
-      estimatedReadiness: ""
+      assigneeIds: [],
+      userRole: undefined,
+      estimatedDuration: undefined,
+      estimatedReadiness: undefined,
+      attachmentUrls: []
     });
     onClose();
   };
 
   /**
-   * Renders task data select dropdown picker with multiple select option
+   * Guesses attachment type from url
```

Comment:
Guesses is funny, perhaps we could say "infers" for maximum professional language but I personally prefer guesses!